### PR TITLE
Advice io1 storage / EC2 I3 instances for AWS

### PR DESCRIPTION
**Pull request by Simran-B**
Please check the wording and if this is sufficient information

### CODE REVIEW

File: 3.5/deployment-cloud-aws.md [hunk 1]
```diff
@@ -20,6 +21,12 @@ or using [Kubernetes](deployment-kubernetes.html),
...
 
-**Important:** In order to deploy on AWS, general guidelines, like using a fast,
-**direct-attached**, SSD disk for the data directory of the ArangoDB processes
-apply.
+{% hint 'info' %}
+In order to deploy on AWS, general guidelines like using a fast, **direct-attached**
+SSD disk for the data directory of the ArangoDB processes apply.
+
+We recommend [EC2 I3 instances](https://aws.amazon.com/ec2/instance-types/i3/){:target="_blank"}
+and to use **io1** storage rather than **gp2** on AWS because of the better throughput
```
Comment (kvahed) [inline 3.5/deployment-cloud-aws.md]: Suggest Instead:

... AWS, which are specifically designed with higher IOPS demands of databases in mind. Please keep in mind that replication by itself generates a manifold of the incoming write load in the back end.

**Comment by Simran-B** [comment]
https://deploy-preview-100--zealous-morse-14392b.netlify.com/docs/3.4/deployment-cloud-aws.html

File: 3.4/deployment-cloud-aws.md [hunk 2]
```diff
@@ -1,25 +1,32 @@
...
-ArangoDB can be deployed on AWS or other cloud platforms. Deploying on a cloud
-provider is common choice and many of the most big ArangoDB installation are running
-on the cloud.
+ArangoDB can be deployed on AWS or other cloud platforms. AWS is the
+infrastructure provider choice for some of the largest ArangoDB installations.
 
-Up to ArangoDB 3.2, official ArangoDB AMI were available in the [AWS marketplace](https://aws.amazon.com/marketplace/search/results/ref=dtl_navgno_search_box?page=1&searchTerms=arangodb){:target="_blank"}.
+Up to and including ArangoDB 3.2, official ArangoDB AMI were available in the
+[AWS marketplace](https://aws.amazon.com/marketplace/search/results/ref=dtl_navgno_search_box?page=1&searchTerms=arangodb){:target="_blank"}.
 Such AMI are not being maintained anymore, though. However, deploying on AWS is
```
Comment (joerg84) [inline 3.4/deployment-cloud-aws.md]: S/AMI/AMIs

File: 3.4/deployment-cloud-aws.md [hunk 3]
```diff
@@ -1,25 +1,32 @@
...
 
 After having initialized your preferred AWS instance, with one of the ArangoDB supported
 operating systems, using the [ArangoDB Starter](deployment-arango-dbstarter.html),
 performing a [Manual Deployment](deployment-manually.html),
-or using [Kubernetes](deployment-kubernetes.html), 
-are all valid options to deploy on AWS. Please refer to the corresponding chapters for further 
-information.
+or using [Kubernetes](deployment-kubernetes.html),
+are all valid options to deploy on AWS. Please refer to the corresponding
+chapters for further information.
```
Comment (joerg84) [inline 3.4/deployment-cloud-aws.md]: Can we add a link?

Comment (Simran-B) [reply 3.4/deployment-cloud-aws.md]: This seems to refer to the three links just above, therefore I removed this sentence as it doesn't add any value.

**Pull request merged**